doc/nrf/config_and_build/dfu/index.rst

```diff
@@ -0,0 +1,49 @@
+.. _app_dfu:
+.. _ug_fw_update:
+
+Device Firmware Updates
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+DFU is...
```

Comment:
Here is what I wrote for the upcoming DevAcademy guide on bootloaders: https://nordicsemi.atlassian.net/wiki/spaces/MAR/pages/287409145/Device+Firmware+Update+DFU+basics 
It has not been reviewed yet, so take it with a pinch of salt.
Also from my above comment, I tried to explain DFU briefly:
"A bootloader and Device Firmware Updates (DFU) are closely related, but essentially are different functionalities. The bootloader is a firmware image, and has two main jobs. First, it verifies the next firmware image, such as the application. Secondly, the bootloader can choose which firmware image to boot next, which is useful when updating the application as part of DFU. DFU is a procedure in two steps. First a new firmware image is transferred to the chip, and then the bootloader tests and boots the new firmware. "